### PR TITLE
Drop data-value logic

### PR DESCRIPTION
**Pull request by orangejenny**
Following up on https://github.com/dimagi/commcare-hq/pull/10929#issuecomment-200966576 since the data-value logic has now also caused a bug (https://github.com/dimagi/commcare-hq/pull/11126)

@calellowitz

### CODE REVIEW

File: corehq/apps/groups/templates/groups/group_members.html [hunk 1]
```diff
@@ -222,9 +222,9 @@
                     <div class="control-group">
                         <label class="control-label" for="group-reporting-input">{% trans "Reporting" %}</label>
                         <div class="controls">
-                            <select class="input-small" name="reporting" id="group-reporting-input" data-value="{{ group.reporting|BOOL }}">
+                            <select class="input-small" name="reporting" id="group-reporting-input">
                                 <option value="true">{% trans "On" %}</option>
```
Comment (calellowitz) [inline corehq/apps/groups/templates/groups/group_members.html]: Is there a reason this select and the one above it never default to true but the one in `module_view_careplan.html` does

**Comment by calellowitz** [comment]
1 question but otherwise looks good to me

File: corehq/apps/groups/templates/groups/group_members.html (hunk 1 above)
Comment (orangejenny) [inline corehq/apps/groups/templates/groups/group_members.html]: This file is correct - the first option doesn't need to be marked `selected`, it gets selected if nothing else is. I'll update module_view_careplan.html to behave the same.

**Comment by calellowitz** [comment]
:+1: on :white_check_mark:

File: corehq/apps/app_manager/templates/app_manager/module_view.html [hunk 2]
```diff
@@ -245,10 +247,9 @@
                         {% trans "Menu Mode" %}
                     </label>
                     <div class="col-sm-4">
-                        <select type="text" name="put_in_root" class="form-control"
-                                data-value="{% if module.put_in_root %}true{% else %}false{% endif %}">
+                        <select type="text" name="put_in_root" class="form-control">
                             <option value="false">{% trans "Display module and then forms" %}</option>
```
Comment (snopoke) [inline corehq/apps/app_manager/templates/app_manager/module_view.html]: This option needs the inverse `selected` logic

Comment (snopoke) [inline corehq/apps/app_manager/templates/app_manager/module_view.html]: Unless you're depending on the default selection?

Comment (snopoke) [inline corehq/apps/app_manager/templates/app_manager/module_view.html]: Read you're comment below :+1:

**Pull request merged**